libs/s25main/ingameWindows/IngameWindow.cpp

```diff
@@ -251,12 +247,15 @@ void IngameWindow::Draw_()
     glArchivItem_Bitmap* rightUpperImg = LOADER.GetImageN("resource", 37);
     rightUpperImg->DrawFull(GetPos() + DrawPoint(GetSize().x - rightUpperImg->getWidth(), 0));
 
-    // The 2 buttons
-    constexpr std::array<helpers::EnumArray<uint16_t, ButtonState>, 2> ids = {{{47, 55, 50}, {48, 56, 52}}};
+    // The buttons
+    using ButtonStateResIds = helpers::EnumArray<unsigned, ButtonState>;
+    constexpr ButtonStateResIds closeResIds = {47, 55, 51};
+    constexpr ButtonStateResIds minimizeResIds = {48, 56, 52};
     if(closeBehavior_ != CloseBehavior::Custom)
-        LOADER.GetImageN("resource", ids[0][buttonState[0]])->DrawFull(GetPos());
+        LOADER.GetImageN("resource", closeResIds[buttonStates_[IwButton::Close]])->DrawFull(GetPos());
```

Comment:
FTR: This could make use of `GetButtonBounds` too to avoid duplicating logic, but not worth changing it at this point.

Reply:
I also like short code but more important is that it is visibly correct. And the 2 similar but different calls next to each other made me need to go to another place in the file to verify this is correct and not a C&P error.   
Hope that was understandable ;-)

It is enough to change it in #1606 for consistency as this is almost merged.

Reply:
> Hope that was understandable ;-)

Sure. ;-)